packages/concerto-cli/index.js

```diff
@@ -54,6 +54,28 @@ require('yargs')
                 console.log(err.message + ' ' + err);
             });
     })
+    .command('saveExternal', 'save local copies of external model dependencies', (yargs) => {
```

Comment:
@dselman and I had suggested `concerto get` which I think is what the `go` compiler uses. https://golang.org/cmd/go/

Other options might be `concerto install` `concerto download` ...